stake-pool/cli/src/main.rs

```diff
@@ -875,15 +876,16 @@ fn command_withdraw(
 
     // Go through prepared accounts and withdraw/claim them
     for withdraw_stake in withdraw_from {
-        let withdraw_amount = pool_data
-            .calc_pool_withdraw_amount(withdraw_stake.amount)
-            .unwrap()
-            + 1;
+        // Convert pool tokens amount to lamports
+        let sol_withdraw_amount = pool_data
+            .calc_lamports_amount(withdraw_stake.pool_amount)
+            .unwrap();
+
         println!(
             "Withdrawing from account {}, amount {} SOL, {} pool tokens",
             withdraw_stake.pubkey,
-            lamports_to_sol(withdraw_stake.amount),
-            lamports_to_sol(withdraw_amount),
+            lamports_to_sol(sol_withdraw_amount),
```

Comment:
Consider using the [Sol](https://github.com/solana-labs/solana/blob/3355efbb5ac0c8a1be23befa68a91da098abc851/sdk/program/src/native_token.rs#L17) type and it'll automatically add the fancy ◎ prefix
```suggestion
            solana_sdk::native_token::Sol(sol_withdraw_amount),
```